recipes/udunits2/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+if [[ $(uname) != Darwin ]]; then
+    yum install -y automake libtool texinfo
```

Comment:
The manylinux Docker image [does include a number of build tools](https://github.com/pypa/manylinux/blob/3a51d5cece16c432cc4d7097414ebbc1da7eec14/docker/build_scripts/build.sh#L34-L40) including `automake` but not `libtool` is included.